src/pages/blog2/sections/BlogBody.jsx

```diff
@@ -7,23 +7,23 @@ import { glovo } from "../../../assets/images/blogs-page";
 import NextRead from "./NextRead";
 import RelatedBlogs from "./RelatedBlogs";
 
-import "./blogBody.css";
-
 function BlogBody({ id, categoryId, blogBody }) {
   const BodyRef = useRef(null);
 
   useEffect(() => {
     if (BodyRef.current) {
-      BodyRef.current.innerHTML += blogBody ?? "";
+      BodyRef.current.innerHTML = blogBody ?? "";
     }
   }, [blogBody]);
 
   return (
     <div className="flex flex-col md:flex-row gap-5 w-full pb-8 px-3">
       <div className="w-full md:w-[70%] flex flex-col gap-5">
         <div className="container text-[13px] md:text-base font-normal text-[#323433]">
-          <div ref={BodyRef} className="blog-content" />
-
+          <div
+            ref={BodyRef}
+            className="prose-lg prose prose-a:no-underline  prose-li:marker:text-[#333] prose-li:marker:text-sm prose-a:text-[#009975]"
+          />
```

Comment:
The text styles are inconsistent with the [original Figma Designs](https://www.figma.com/design/QZwtseXogSFjLNz64uLwjV/SYT-Website-Redesign-Complete?node-id=6943-12373&m=dev) i.e it's too small for a blog post and the color is too light.

Reply:
Thanks for adding the typography, that was a good call.